prow/cmd/deck/static/tide.html

```diff
@@ -34,7 +34,7 @@ <h4>Your Pull Request must also match one of these GitHub search queries (you ca
                 <th>Batch</th>
                 <th>Passing</th>
                 <th>Pending</th>
-                <th>Missing</th>
+                <th>Failing/Untested against HEAD</th>
```

Comment:
`HEAD` is technically a clone-local reference and even the more general `HEAD` meaning `master` isn't correct when the PR targets a non-`master` branch so maybe just `"Failing or Untested"` as a header?

Reply:
Ok, that makes sense.
I'm thinking that the case in which a PR has all green status checks against an older commit could still be confusing since the PR will appear to be both tested and passing.  Thats probably too much information to convey in a column header though. It probably makes more sense to include that at the top of the Tide dashboard. Maybe as part an addition to:
> All GitHub statuses on a Pull Request must be passing / green for automatic merge ✓